web/src/main/java/it/pagopa/selfcare/party/registry_proxy/web/controller/GeographicTaxonomiesController.java

```diff
@@ -32,28 +33,48 @@ public GeographicTaxonomiesController(GeographicTaxonomiesService geographicTaxo
      */
     @ResponseStatus(HttpStatus.OK)
     @ApiOperation(value = "${swagger.registry-proxy.api.geotaxonomies.getGeographicTaxonomiesByDescription}", notes = "${swagger.registry-proxy.api.geotaxonomies.getGeographicTaxonomiesByDescription}")
-    @GetMapping(value = "", produces = MediaType.APPLICATION_JSON_VALUE)
+    @GetMapping(value = "/geotaxonomies", produces = MediaType.APPLICATION_JSON_VALUE)
     public ResponseEntity<List<GeographicTaxonomyResource>> retrieveGeoTaxonomiesByDescription(@ApiParam("${swagger.api.geotaxonomy.model.description}")
-                                                                                         @RequestParam(value = "description") String description,
+                                                                                               @RequestParam(value = "description") String description,
                                                                                                @ApiParam("${swagger.api.geotaxonomy.model.offset}")
-                                                                                         @RequestParam(value = "offset", required = false) Integer offset,
+                                                                                               @RequestParam(value = "offset", required = false) Integer offset,
                                                                                                @ApiParam("${swagger.api.geotaxonomy.model.limit}")
-                                                                                         @RequestParam(value = "limit", required = false) Integer limit) {
+                                                                                               @RequestParam(value = "limit", required = false) Integer limit) {
 
         CustomExceptionMessage.setCustomMessage(RETRIEVE_GEO_TAXONOMIES_ERROR);
         List<GeographicTaxonomy> geographicTaxonomyList = geographicTaxonomiesService.retrieveGeoTaxonomiesByDescription(description, offset, limit);
-        return ResponseEntity.ok().body(GeographicTaxonomyMapper.toResource(geographicTaxonomyList));
+        return ResponseEntity.ok().body(GeographicTaxonomyMapper.toResourceList(geographicTaxonomyList));
+    }
+
+
+    /**
+     * The function returns geographic taxonomy by code
+     *
+     * @param code String
+     * @return List
+     * * Code: 200, Message: successful operation, DataType: GeographicTaxonomies
+     * * Code: 404, Message: GeographicTaxonomies not found, DataType: Problem
+     */
+    @ResponseStatus(HttpStatus.OK)
+    @ApiOperation(value = "${swagger.registry-proxy.api.geotaxonomies.getGeographicTaxonomyByCode}", notes = "${swagger.registry-proxy.api.geotaxonomies.getGeographicTaxonomyByCode}")
+    @GetMapping(value = "/geotaxonomy/{geotaxId}", produces = MediaType.APPLICATION_JSON_VALUE)
+    public GeographicTaxonomyResource retrieveGeoTaxonomiesByCode(@ApiParam("${swagger.geographicTaxonomy.model.geotaxId}")
```

Comment:
il path corretto sarebbe geotaxonomies/{geotaxId} stai andando a recuperare una tassonomia specifica da una lista di tassonomie e lo recuperi tramite l'id specifico
in più fai attenzone al nome del metodo retrieveGeoTaxonomiesByCode, sembra che ne recuperi più di uno.
il v1/geotaxonomies deve stare come basePath, nel RequestMapping
mentre il /{geotaxId} è quello che va nella get